frontend/src/main/components/Rider/RiderTable.js

```diff
@@ -0,0 +1,87 @@
+import React from "react";
+import OurTable, { ButtonColumn } from "main/components/OurTable";
+import { useBackendMutation } from "main/utils/useBackend";
+import { cellToAxiosParamsDelete, onDeleteSuccess } from "main/utils/riderUtils"
+import { useNavigate } from "react-router-dom";
+
+export default function RiderTable({
+        ride
+    }) {
+
+    const navigate = useNavigate();
+
+    const editCallback = (cell) => {
+        navigate(`/ride_request/edit/${cell.row.values.id}`)
+    }
+
+    // Stryker disable all : hard to test for query caching
+
+    const deleteMutation = useBackendMutation(
+        cellToAxiosParamsDelete,
+        { onSuccess: onDeleteSuccess },
+        ["/api/ride_request/all"]
+    );
+    // Stryker enable all 
```

Comment:
Should be `// Stryker restore all`

See: https://ucsb-cs156-s23.slack.com/archives/C050779M19P/p1685726928710669

Reply:
This has been fixed in the latest commit